aten/src/ATen/native/sparse/SparseMatMul.cpp

```diff
@@ -221,11 +230,22 @@ void sparse_matmul_kernel(
   Tensor output_row_indices = output_indices.select(0, 0);
   Tensor output_col_indices = output_indices.select(0, 1);
 
-  _csr_matmult(M, N, mat1_indptr.data_ptr<int64_t>(), mat1_col_indices.data_ptr<int64_t>(), mat1_values.data_ptr<scalar_t>(),
-  mat2_indptr.data_ptr<int64_t>(), mat2_col_indices.data_ptr<int64_t>(), mat2_values.data_ptr<scalar_t>(),
-  output_indptr.data_ptr<int64_t>(), output_col_indices.data_ptr<int64_t>(), output_values.data_ptr<scalar_t>());
+  // TODO: replace with a CSR @ CSC kernel for better performance.
```

Comment:
This is highly recommended because `CSR @ CSC` is extremely fast compared to other possible schemas.

Reply:
There is no such kernel yet.

Reply:
> The CSR@CSC still converts CSC to CSR

yeah, this is pretty unfortunate, but OT for this PR.